source/Engine/main.cpp

```diff
@@ -257,7 +259,7 @@ void checkInput() {
 					case SDLK_F12:
 						input.keyPressed = 63247; break;
 					default:
-						input.keyPressed = event.key.keysym.unicode;
+						input.keyPressed = event.key.keysym.sym;
```

Comment:
This whole input stuff needs testing on every platform and with every game that takes keyboard input. Works on GNU/Linux with TSoTC for me, but there's very limited keyboard handling, so it needs checking if it's not broken somewhere else.